static/src/js/providers/AuthContextProvider/AuthContextProvider.jsx

```diff
@@ -32,12 +32,16 @@ const AuthContextProvider = ({ children }) => {
   const [cookies, setCookie, removeCookie] = useCookies(['loginInfo', 'launchpadToken', 'data'])
   const { loginInfo = {}, launchpadToken } = cookies
 
-  const setUser = (info) => {
-    setCookie('loginInfo', info)
-  }
+  const setUser = useCallback((arg) => {
```

Comment:
I remember we had this code, looks like somewhere along the line it got reverted.

Reply:
This one is also another fix from Trevor last week with constant API call issue